lighthouse-core/scripts/chromium-web-tests/utils.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
I'd still be in favor of removing all the unused functions if we don't intend to roll this anymore, then we're only left with like 50 lines to have to think about here

Reply:
If we did this should it still go in third party?

Reply:
Aren't we also Chromium contributors? :)

Reply:
The wording in https://opensource.google/docs/releasing/preparing/#third-party-components isn't totally clear since Chromium also uses the Google CLA for contributions, and relicensing makes it an even harder question.

I'd say absent outside advice, we should stick to `third-party/` with the original license (including for modifications on top of the original code).

Reply:
for utils.js specifically if we end up replacing every single function implementation with delegates to our dependencies that's not even "modifications on top of" anything, it's a replacement, but if we're looking at some hybrid of just trimming the fat then I could see the argument for living in third-party

Reply:
of course if we're keeping all of download anyway then we might as well move utils to live there too :)